contents/projectManagement/githubActions.md

```diff
@@ -0,0 +1,155 @@
+<frontmatter>
+  title: Workflow Automation with GitHub Actions
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# GitHub Actions
+
+📝 [Alfred Yip](https://github.com/alyip98)
+
+🔎 [James Pang](https://github.com/jamessspanggg), [Tan Yuanhong](https://github.com/le0tan), [Tejas Bhuwania](https://github.com/Tejas2805)
+
+
+<box id="article-toc">
+
+* [What is GitHub Actions‎](#what-is-github-actions)
+* [Why GitHub Actions‎](#why-github-actions)
+* [How does GitHub Actions Work‎](#how-does-github-actions-work)
+* [How is GitHub Actions Being Used‎](#how-is-github-actions-being-used)
+* [How to Get Started with GitHub Actions‎](#how-to-get-started-with-github-actions)
+* [Where to Go From Here‎](#where-to-go-from-here)
+</box>
+
+## What is GitHub Actions
+<!--Explain how X fits into the big picture of SE. Describe it relative to topics the reader is likely to know.-->
+GitHub Actions is a workflow automation solution that is tightly integrated into the GitHub ecosystem. It is commonly used to provide end-to-end continuous integration and continuous deployment (CI/CD) directly in your repository. With its tight integration with GitHub's ecosystem of services, users have the potential to automate more than just CI/CD.
```

Comment:
tight integration mentioned twice in the space of two sentences. Can do with just one.